data_processors/pipeline/orchestration/rnasum_step.py

```diff
@@ -140,6 +140,9 @@ def prepare_rnasum_jobs(this_workflow: Workflow) -> List[Dict]:
     # Get patient specific reference dataset
     tumor_dataset = lookup_tcga_dataset(meta=meta_tumor)
 
+    # Extend umccrise directory by one additional directory that is named as '<subject_id>__<sample_name>'
```

Comment:
Is the
```
umccrise_directory = liborca.parse_umccrise_workflow_output_directory(this_workflow.output)
```
definition better suited to be here since it's the only place it's being used?

Reply:
That would likely be better, since other future potential downstream of umccrise workflows could use it too. Will check @victorskl opinion on this

Reply:
Can the sub folder path we're after be extracted or constructed from the workflow output alone?

Looking at the code, that does not seem to happen. If that's not possible I'd be careful not to mix concepts (or compromise the simple task of the parser)?

Reply:
Also, probably need to make sure that the string concatenation has a `/` that separates the two - not sure if `umccrise_directory` will end with `/` below.

Reply:
Ah good, point, my guess is that it won't.

Reply:
Ah, we want a result subdirectory from umccrise output. Okay, in this case, I'd recommend we do: 

- keep the sub-dir string construct as a separate function; say, `rnasum_step.get_umccrise_result_root(...)`
- then, I'd unittest that function in `test_rnasum_step.py` -- to make we bind this contract! :)  -- also test the path format, slashes, etc...

And one day; when elsewhere need this snippet, then I'd refactor it into `liborca`.

How is that sound?

Reply:
> Yep I can do

Yes, pls. And yep, sure; `get_umccrise_sample_results_from_root` sound better!

One more thing is that `meta_tumor` is WTS tumor meta instance at this point. Looks like; if this is meant to be WGS tumor meta then need to extract it from `umccrise_meta_list_df` up there, too. Might get a tad tricky!

Reply:
Should be good to go now

Reply:
Thanks Alexis; let me take over the branch in tick, next. And to add unittest there. All good.

Reply:
Thanks Alexis; let me take over the branch in tick, next. And to add unittest there. All good.